uk.co.compucorp.civicrm.hrleaveandabsences/api/v3/LeavePeriodEntitlement.php

```diff
@@ -79,4 +79,38 @@ function _civicrm_api3_leave_period_entitlement_getremainder_spec(&$params) {
   $params['include_future']['api.required'] = 0;
 }
 
+/**
+ * LeavePeriodEntitlement.getbreakdown API
+ *
+ * @param array $params
+ *
+ * @return array API result descriptor
+ *
+ * @throws API3_Exception
+ */
+function civicrm_api3_leave_period_entitlement_getbreakdown($params) {
+  $hasEntitlementID = !empty($params['entitlement_id']);
+  $hasContactAndPeriodID = !empty($params['contact_id']) && !empty($params['period_id']);
+  $hasContactOrPeriodID = !empty($params['contact_id']) || !empty($params['period_id']);
+  if(($hasEntitlementID && $hasContactAndPeriodID) || (!$hasEntitlementID && !$hasContactAndPeriodID) || ($hasEntitlementID && $hasContactOrPeriodID)) {
```

Comment:
Trying to catch the condition where entitlement_id is present and either of contact or period id is also present. This should not be so according to the ticket.